data_management/phase_tkinter_class.py

```diff
@@ -42,21 +42,21 @@ def __init__(self, next_phase, master=None, prev_phase: tk.Frame = None, test: b
 
             master.update_idletasks()
             # ask the user to specify a file
-            self.filename = fd.askopenfilename(
+            self.os_filename = fd.askopenfilename(
                 initialdir=path.join(path.dirname(__file__), "..", "data", "transcribed_stories", "51--", "5101"))
 
             # Check if the OS is Windows or Linux based
-            if ':' in self.filename: # Windows
+            if ':' in self.os_filename: # Windows
```

Comment:
`if os.name == 'nt':` is the pythonic way of checking if we are running in a windows environment, technically colons are valid on linux for a filename

Reply:
Wow, good spot thank you!